packages/zowe-explorer/CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 - **Breaking:** Zowe Explorer no longer uses a temporary directory for storing Data Sets and USS files. All settings related to the temporary downloads folder have been removed. In order to access resources stored by Zowe Explorer v3, refer to the [FileSystemProvider documentation](https://github.com/zowe/zowe-explorer-vscode/wiki/FileSystemProvider) for information on how to build and access resource URIs. Extenders can detect changes to resources using the `onResourceChanged` function in the `ZoweExplorerApiRegister` class. [#2951](https://github.com/zowe/zowe-explorer-vscode/issues/2951)
 - Implemented the `onVaultUpdate` VSCode events to notify extenders when credentials are updated on the OS vault by other applications. [#2994](https://github.com/zowe/zowe-explorer-vscode/pull/2994)
 - Implemented the `onCredMgrsUpdate` VSCode events to notify extenders when the local PC's credential manager has been updated by other applications. [#2994](https://github.com/zowe/zowe-explorer-vscode/pull/2994)
+- Changed default base profile naming scheme in newly generated configuration files to prevent name and property collisions between Global and Project profiles [#2682](https://github.com/zowe/zowe-explorer-vscode/issues/2682)
```

Comment:
do we need to say the word "collision"? Is this a technical term? I was thinking maybe we could use "conflict," if possible.